vsphere/assets/configuration/spec.yaml

```diff
@@ -292,6 +292,39 @@ files:
           display_default: ['vm', 'host']
           items:
             type: string
+      - name: include_events
+        description: |
+          The events that should be collected by the vSphere integration.
+
+          Each event is an object represented with the event name itself as a key.
```

Comment:
Nit- the event name is not the key since "event" is the key.

Reply:
Fixed.